2주차/23827/23827_Java_짱구.java

```diff
@@ -0,0 +1,31 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st;
+
+        int N = Integer.parseInt(br.readLine());
+        long[] arr = new long[N];
+
+        st = new StringTokenizer(br.readLine());
```

Comment:
```suggestion
        StringTokenizer st = new StringTokenizer(br.readLine());
```
st를 여기서 한번만 선언하는 것 같아서 함께 사용하면 더 좋을 것 같아요 ㅎㅎ